heron/executor/src/python/heron_executor.py

```diff
@@ -155,7 +171,14 @@ def init_parsed_args(self, args):
     # id within docker, rather than the host's hostname. NOTE: this 'HOST' env variable is not
     # guaranteed to be set in all Docker executor environments (outside of Marathon)
     if is_docker_environment():
-      self.master_host = os.environ.get('HOST') if 'HOST' in os.environ else socket.gethostname()
+       # Need to set the HOST environment vairable if docker is for AWS ECS tasks
+      if isEcsAmiInstance():
+        self.master_host = subprocess.Popen(["curl",
+                                             "http://169.254.169.254/latest/meta-data/local-ipv4"]
+                                            , stdout=subprocess.PIPE).communicate()[0]
+        os.environ['HOST'] = self.master_host
```

Comment:
No need for this line since you won't be grabbing `os.environ['HOST']` after this